src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -8,6 +9,8 @@
  */
 public abstract class Command {
 
+    public static final Index TAB_ID = Index.fromOneBased(3);
```

Comment:
Maybe we can have an enum for TAB_ID instead to make it more clear.

Reply:
Sorry, can you explain what is the TAB_ID used for in the Command class actually? Is it only relevant in the CommandResult class?